webview/src/views/history.svelte

```diff
@@ -68,7 +68,7 @@
         <p>Workflow Task ({workflowTask.status})</p>
       </div>
       {#each workflowTask.events as event}
-        <li>
+        <li title={`event ID: ${event.eventId}`}>
```

Comment:
Nit to match casing we have in oss/cloud UI
```suggestion
        <li title={`Event ID: ${event.eventId}`}>
```